Графика для всех страниц

a-pestretsova opened the pull request:
---
:mortar_board: [Графика для всех страниц](https://up.htmlacademy.ru/htmlcss/35/user/2059277/tasks/7)

a-pestretsova committed:
Добавляет папку с изображениями и графику на все страницы

captain-zsa reviewed (commented):
1. В названиях не использовать числа. Это было в статье про именование сущностей а также в статье про абсолютные и относительные пути. К примеру те же отели сейчас – поменяется сортировка и будет вообще непонятно где какой там на каком месте. А если бы хотя бы по названиям было – всё было бы норм и понятно где какая картинка от какого отеля.
2. Не надо разграничивать всё на папки до такой степени сильно. Завтра фото из каталога потребуются на какой-то ещё странице. Или иконки потребуются на какой-то другой странице и что делать?

a-pestretsova committed:
Вносит правки по графике

Также здесь подключаются стили на страницах, начата работа по базовой стилизации

captain-zsa reviewed (commented):
greenTree большая буква. Может у тебя и маленькая на компе но гитхаб у файлов такие моменты увы не отслеживает. Придется переименовать иначе как-то. Можно через дефис, тогда он подтянет изменения.


У тебя еще есть сердечко с кружком и числом. Не нужно это. Как мы в кружке число менять то будем? Не получится это делать если число будет внутри картинки, поэтому данный элемент нужно будет верстать отдельно.

звездочки все удалить кроме одной. Мы будем их через background вставлять, а у него есть возможность повторять картинку по X и по Y если это нужно будет (нам по X нужно будет как раз).

всё, больше комментов не будет)) это последнее что надо поправить.

a-pestretsova committed:
Вносит последние правки по графике

a-pestretsova committed:
Добавляет базовую стилизацию главной страницы
